src/Serilog.Sinks.PeriodicBatching/Serilog.Sinks.PeriodicBatching.csproj

```diff
@@ -20,7 +20,7 @@
     <RepositoryType>git</RepositoryType>
     <GenerateAssemblyVersionAttribute>false</GenerateAssemblyVersionAttribute>
     <!-- Don't reference the full NETStandard.Library -->
-    <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>    
+    <DisableImplicitFrameworkReferences Condition=" '$(TargetFramework)' == 'net45' ">true</DisableImplicitFrameworkReferences>
```

Comment:
I'm not sure this does anything at all on .NET Framework; can probably drop it completely, now.

Reply:
@nblumhardt It definitely does _something_:

![image](https://user-images.githubusercontent.com/28367120/96753925-6cb03f00-1385-11eb-9300-d2432986d489.png)

Whether or not that _something_ is important, not sure 😄 Happy to remove if you want, just please confirm for me.

Reply:
:-) on .NET Framework, these assemblies will be in the GAC, so there's no packaging overhead like there would have been for the earlier netstandard targets - should be "nothing important" 👍

Reply:
Updated